### PR TITLE
[WIP] Content improvements

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
- [x] geïntegreerd
- [x] change u to je in nl.yml
- [x] change u to je on starterkit page

### CODE REVIEW

File: locales/nl.yml [hunk 1]
```diff
@@ -41,7 +41,7 @@ nl:
       title: "Opleiden & Leren met CAPP LMS"
     name: CAPP
   contact:
-    form_title: "Wat kunnen we voor je betekenen?"
+    form_title: "Wat kunnen we voor u betekenen?"
```
Comment (fatboypunk) [inline locales/nl.yml]: why are we going back to `u`?

Comment (RonaldDijkstra) [inline locales/nl.yml]: Wouldn't say back. There's no consistency, we use both je and u in different parts of the website.

Comment (fatboypunk) [inline locales/nl.yml]: yeah we intended to use `je` and `jouw`

Comment (dkhgh) [inline locales/nl.yml]: Rather not

Sent from my iPhone

On 19 Mar 2016, at 10:26, Marcel Horlings <notifications@github.com<mailto:notifications@github.com>> wrote:


In locales/nl.yml<https://github.com/DefactoSoftware/website/pull/178#discussion_r56746743>:

> @@ -41,7 +41,7 @@ nl:
>        title: "Opleiden & Leren met CAPP LMS"
>      name: CAPP
>    contact:
> -    form_title: "Wat kunnen we voor je betekenen?"
> +    form_title: "Wat kunnen we voor u betekenen?"


why are we going back to u?

-

Comment (dkhgh) [inline locales/nl.yml]: Yep

Sent from my iPhone

On 19 Mar 2016, at 11:10, Marcel Horlings <notifications@github.com<mailto:notifications@github.com>> wrote:


In locales/nl.yml<https://github.com/DefactoSoftware/website/pull/178#discussion_r56747207>:

> @@ -41,7 +41,7 @@ nl:
>        title: "Opleiden & Leren met CAPP LMS"
>      name: CAPP
>    contact:
> -    form_title: "Wat kunnen we voor je betekenen?"
> +    form_title: "Wat kunnen we voor u betekenen?"


yeah we intended to use je and jouw

-

Comment (dkhgh) [inline locales/nl.yml]: I also prefer 'je'

Comment (RonaldDijkstra) [inline locales/nl.yml]: I prefer consistency, so will change the 'u's' to 'je's'. Thanks for feedback.

**Comment by RonaldDijkstra** [comment]
Wrong commit message on 9c78304, I meant change 'u' for 'je'.

**Comment by fatboypunk** [comment]
Needs a rebase

**Comment by RonaldDijkstra** [comment]
I abandon this and make the changes on page level PR's.

**Comment by fatboypunk** [comment]
So this can be closed?

**Comment by RonaldDijkstra** [comment]
Yes, I'll use it as reference.

**Pull request closed without merging**